### PR TITLE
Assign openshift sdn traffic to system priority level

### PR DESCRIPTION
**Pull request by tkashem**
Assign sdn traffic to `system` priority level, this will bring sdn traffic to the same priority level as kubelet traffic. 

Currently the kube-apiserver does not allocate the sdn traffic to any dedicated concurrency pool, so treating it no different than the default cluster workload. This change will allow the kube-apiserver to allocate the sdn traffic to a known priority level `system` shared by kubelet.

For more information on priority and fairness - https://kubernetes.io/blog/2020/04/06/kubernetes-1-18-feature-api-priority-and-fairness-alpha/

### CODE REVIEW

File: manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      clusterScope: true
+      namespaces:
+      - '*'
+      resources:
+      - '*'
+      verbs:
+      - '*'
+    subjects:
+    - kind: ServiceAccount
+      serviceAccount:
```
Comment (tkashem) [inline manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml:31]: did i get the service account right?

Comment (squeed) [reply manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml:31]: We have two service accounts, `sdn-controller` and `sdn`. They're both system priority.

**Comment by squeed** [comment]
One small change (both SAs) but otherwise seems reasonable.

File: manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml (hunk 1 above)
Comment (tkashem) [reply manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml:31]: @squeed I added `sdn-controller` as you suggested.

**Comment by squeed** [comment]
Looks reasonable
/approve
I'll leave lgtm to @deads2k, since I can't speak for apiserver QOS.

**Comment by deads2k** [comment]
@tkashem this is probably worth a backport.

**Comment by dcbw** [comment]
@tkashem can you rebase? Thanks!

**Comment by tkashem** [comment]
@dcbw rebase completed, it needs re-lgtm.

**Comment by danwinship** [comment]
Um, this seems wrong; the `sdn` and `sdn-controller` ServiceAccounts are only created if openshift-sdn is selected as the network plugin, so this will be referring to non-existent ServiceAccounts when ovn-kubernetes, kuryr, or a third-party network plugin is in use. Even if that doesn't make things fail, we definitely want this traffic priority thing to be applying at least to both openshift-sdn and ovn-kubernetes...

**Comment by tkashem** [comment]
> Um, this seems wrong; the sdn and sdn-controller ServiceAccounts are only created if openshift-sdn is selected as the network plugin, so this will be referring to non-existent ServiceAccounts when ovn-kubernetes, kuryr, or a third-party network plugin is in use. Even if that doesn't make things fail, we definitely want this traffic priority thing to be applying at least to both openshift-sdn and ovn-kubernetes...

@danwinship so this implies that we have to create a flowschema based on the network stack installed by the network operator.

**Comment by squeed** [comment]
Yes, @danwinship is right. I shouldn't have approved this.

**Pull request merged**